doc/classes/MenuButton.xml

```diff
@@ -27,6 +27,11 @@
 			</description>
 		</method>
 	</methods>
+	<members>
+		<member name="switch_on_hover" type="bool" setter="set_switch_on_hover" getter="is_switch_on_hover">
+			If [code]true[/code], when the cursor hovers above another MenuButton that [code]switch_on_hover[/code] is also enabled, within the same parent, it will close the current MenuButton, opening the one that's being hovered.
```

Comment:
I think this might be better if slightly restated:
If `true`, when the cursor hovers above another MenuButton  within the same parent **which also has** `switch_on_hover` enabled, it will close the current MenuButton **and open** the **other** one.